README.md

```diff
@@ -120,6 +119,8 @@ For more information about populating database, see the [Populating the Database
         - Replace the `$DOMAIN` parameter in `nginx.conf` with the domain name (excluding the `http://` part, eg. `www.domain.com`) where you plan to host the site.
```

Comment:
can you refer https://github.com/elasticpath/react-pwa-reference-storefront/pull/131/files?short_path=d0a94c7#diff-d0a94c79b84b081325ed51136925e08a and rewrite the section ?

Reply:
Thanks for calling this out. There's actually some incorrect statements in the tutorial which I'll clean up as well now.